### PR TITLE
Rm empty workflows

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
In #2744 the aggregator and docker build tests were consolidated into build.yaml.  The aggregator and docker build files were kept as placeholders, to pass the CI merge requirements.  Those files have been removed as merge requirements now, so the placeholder files are no longer needed.

# Changes
* Delete placeholder aggregator and docker-build workflows.

# Tests
None

# Todos

- [ ] Add entry to changelog (if necessary).
  - Not needed as this does not affect the wasm in any way.

### CODE REVIEW

File: .github/workflows/aggregator.yaml
Comment (lmuntaner) [inline .github/workflows/aggregator.yaml:1]: The changelog is not only for wasm related changes.

It has two entries: "Application" and "Operations".

These changes go into "Operations". I would add also the consolidation in the build.yaml, not just the removal if you don't mind.

**Review by lmuntaner — APPROVED** [review]
Approving, still pending the Changelog though

File: .github/workflows/aggregator.yaml
Comment (bitdivine) [reply .github/workflows/aggregator.yaml:1]: Sure thing.

Comment (bitdivine) [reply .github/workflows/aggregator.yaml:1]: Done.  Do you want to check again or should I merge?

Comment (lmuntaner) [reply .github/workflows/aggregator.yaml:1]: No need to wait for me next time.

**Pull request merged**